<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiveWEClass</class>
 <widget class="QMainWindow" name="HiveWEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiveWE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GLWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionTest_Map"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuNew_Pallete">
     <property name="title">
      <string>New Pallete</string>
     </property>
     <addaction name="actionPathing_Pallete"/>
    </widget>
    <addaction name="menuNew_Pallete"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDoodads"/>
    <addaction name="actionPathing"/>
    <addaction name="actionBrush"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTileSetter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionConvert_Blockers_to_Pathmap">
   <property name="text">
    <string>Convert Blockers to Pathing Map</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Map...</string>
   </property>
  </action>
  <action name="actionConvert_Tile_Type_to_PathMap">
   <property name="text">
    <string>Convert Tile Type to Pathing Map</string>
   </property>
  </action>
  <action name="actionPathing_Pallete">
   <property name="text">
    <string>Pathing</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Map</string>
   </property>
  </action>
  <action name="actionTest_Map">
   <property name="text">
    <string>Test Map</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save Map As...</string>
   </property>
  </action>
  <action name="actionDoodads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Doodads</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionPathing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pathing</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionTileSetter">
   <property name="text">
    <string>Modify Tileset...</string>
   </property>
  </action>
  <action name="actionTileReplace">
   <property name="text">
    <string>Replace Tile...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>GLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HiveWE.qrc"/>
 </resources>
 <connections/>
</ui>
